boxsdk/auth/oauth2.py

```diff
@@ -28,6 +28,7 @@ def __init__(
             access_token=None,
             refresh_token=None,
             network_layer=None,
+            retrieve_tokens=None,
```

Comment:
Since this is similar to store_tokens (and they'll probably be used together), let's put this below store_tokens in the argument list. Same in the docstring.